libs/wingsdk/src/std/resource.ts

```diff
@@ -137,7 +185,7 @@ export abstract class Resource extends Construct implements IResource {
    *
    * @internal
    */
-  public static _registerOnLiftObject(
+  public static _onLiftObject(
```

Comment:
Does this need to be on `Resource` and internally exposed or can this just be a private utility function somewhere?

Reply:
Moved